### PR TITLE
INTYGFV-15367: Modify columns

### PR DESCRIPTION
**Pull request by SamuelTilly**

### CODE REVIEW

**Review by UlrikaSopra — COMMENTED** [review]
Great job!! 👯  just a small thing, if you have chosen not to show personal data, the social security number and name etc. are still in the list (= Maybe a question for Rebecka.

image

File: apps/rehabstod/src/pages/CurrentSickLeaves/CurrentSickLeaves.tsx [hunk 1]
```diff
@@ -53,18 +53,26 @@ export function CurrentSickLeaves() {
...
+          />
+        </div>
+
+        <div className="mb-5 w-96 self-end">
+          <ModifySicknessTableColumns />
+        </div>
+      </div>
 
-      <Table column={SickLeaveColumn.Startdatum}>
+      <Table sortColumn={SjukfallColumn.Startdatum}>
```
Comment (mhornfeldt) [inline apps/rehabstod/src/pages/CurrentSickLeaves/CurrentSickLeaves.tsx:75]: We should use the term SickLeave instead of Sjukfall

**Review by mhornfeldt — CHANGES_REQUESTED** [review]
Change Sjukfall to SickLeave

**Comment by SamuelTilly** [comment]
> Change Sjukfall to SickLeave

Good call, has been changed to SickLeave

**Review by mhornfeldt — APPROVED** [review]

**Pull request merged**